tests/snappi/test_vlan_traffic.py

```diff
@@ -0,0 +1,169 @@
+                "$PORT_2",
+                "SAI_BRIDGE_PORT_ATTR_ADMIN_STATE",
+                "true",
+                "SAI_BRIDGE_PORT_ATTR_FDB_LEARNING_MODE",
+                "SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW",
+            ],
+        },
+        {
+            "name": "vlan_member_2",
+            "op": "create",
+            "type": "SAI_OBJECT_TYPE_VLAN_MEMBER",
+            "attributes": [
+                "SAI_VLAN_MEMBER_ATTR_VLAN_ID",
+                "$vlan_10",
+                "SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID",
+                "$bridge_p_2",
+                "SAI_VLAN_MEMBER_ATTR_VLAN_TAGGING_MODE",
+                "SAI_VLAN_TAGGING_MODE_UNTAGGED",
+            ],
+        },
+        {
+            "name": "PORT_2",
+            "op": "set",
+            "type": "SAI_OBJECT_TYPE_PORT",
+            "attributes": [
+                "SAI_PORT_ATTR_ADMIN_STATE",
+                "true",
+                "SAI_PORT_ATTR_PORT_VLAN_ID",
+                "10",
+            ],
+        },
+        {
+            "name": "bridge_p_3",
+            "op": "create",
+            "type": "SAI_OBJECT_TYPE_BRIDGE_PORT",
+            "attributes": [
+                "SAI_BRIDGE_PORT_ATTR_TYPE",
+                "SAI_BRIDGE_PORT_TYPE_PORT",
+                "SAI_BRIDGE_PORT_ATTR_PORT_ID",
+                "$PORT_3",
+                "SAI_BRIDGE_PORT_ATTR_ADMIN_STATE",
+                "true",
+                "SAI_BRIDGE_PORT_ATTR_FDB_LEARNING_MODE",
+                "SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW",
+            ],
+        },
+        {
+            "name": "vlan_member_3",
+            "op": "create",
+            "type": "SAI_OBJECT_TYPE_VLAN_MEMBER",
+            "attributes": [
+                "SAI_VLAN_MEMBER_ATTR_VLAN_ID",
+                "$vlan_10",
+                "SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID",
+                "$bridge_p_3",
+                "SAI_VLAN_MEMBER_ATTR_VLAN_TAGGING_MODE",
+                "SAI_VLAN_TAGGING_MODE_UNTAGGED",
+            ],
+        },
+        {
+            "name": "PORT_3",
+            "op": "set",
+            "type": "SAI_OBJECT_TYPE_PORT",
+            "attributes": [
+                "SAI_PORT_ATTR_ADMIN_STATE",
+                "true",
+                "SAI_PORT_ATTR_PORT_VLAN_ID",
+                "10",
+            ],
+        },
+    ]
+    for command in commands:
+        print("-" * 80)
+        print(command)
+        result = npu.command_processor.process_command(command)
+        print(result)
+
+    macs = ["00:11:11:11:11:11", "00:22:22:22:22:22"]
+    config = dataplane.configuration
+    config.options.port_options.location_preemption = True
+    layer1 = config.layer1.layer1()[-1]
+    layer1.name = "port settings"
+    layer1.port_names = [port.name for port in config.ports]
+    layer1.ieee_media_defaults = False
+    layer1.auto_negotiation.rs_fec = False
+    layer1.auto_negotiation.link_training = False
+    layer1.speed = "speed_100_gbps"
```

Comment:
Can the test be written in such a way (e.g. setup file) to select line rate based on type of traffic generator? We've been advertising this is the case. Of course not *every* test can run on either. But perhaps line rate can come from the setup file?

Reply:
@selldinesh , is this already addressed? please confirm

Reply:
we have discussed on this regard and agreed not change the script and keep it as it is.